e2e/LongHaul/device/MessageHelpers.cs

```diff
@@ -11,27 +11,24 @@ internal static class MessageHelpers
         /// </summary>
         public const int MaxMessagePayloadBytes = (256 * 1024) - 1024;
 
-        private const int Iso8601Length = 24;
+        public static int GetMessagePayloadSize(TelemetryMessage message)
```

Comment:
It seems that we don't have any references in this class, but upgraded to V2 contexts just in case.

Reply:
The original purpose of this was to enable sending messages in bulk.

We should only use it for AMQP, because MQTT doesn't do true bulk sending.